op-bindings/bindings/weth9.go

```diff
@@ -31,7 +31,7 @@ var (
 }
```

Comment:
The old weth9 has:
- bed565cbea90624f5b56e2f56c4d5fc05493e40220bba9bd15ace6aa4cabb51c
The new weth9 has:
- 2282f823392ab6b42c17b31235784e88e108cd8d10e77cd732aa3e1e7a88a7d

This looks like a bytecode hash

The diff is right at the very end